src/systems/gltf-animations.js

```diff
@@ -0,0 +1,31 @@
+var registerSystem = require('../core/system').registerSystem;
+var THREE = require('../lib/three');

```

Comment:
That'll be a handy feature when/if I/someone adds extra animation features (like from the component, this.system.addAnimation(animation); and in system, function playAllAnimations() { /* loop over this.animations */ }